app/api/routes.py

```diff
@@ -465,6 +479,29 @@ def update_resource(id, json, db):
         return utils.standardize_response(status_code=500)
 
 
+def create_resources(json, db):
+    try:
+        created_resources = []
+
+        for resource in json:
+            res = create_resource(resource, db)
+
+            # Pass along standardized responses generated from create_resource
+            if isinstance(res, tuple):
+                return res
```

Comment:
I am having trouble understanding this part, why are we returning `res` here?

Reply:
If an error was produced, it will be stored in `res` as a tuple. We want to pass that along so `standardize_response` can parse it and pass it along to the client. The assumption is that the only situation in which a resource would be a tuple rather than a dictionary is if there was an error. Though I agree it makes readability a little difficult. Open to suggestions on a better way to do this.